public/app/features/correlations/components/Table/index.tsx

```diff
@@ -0,0 +1,173 @@
+import { cx, css } from '@emotion/css';
+import React, { useMemo, Fragment, ReactNode } from 'react';
+import {
+  CellProps,
+  SortByFn,
+  useExpanded,
+  useSortBy,
+  useTable,
+  DefaultSortTypes,
+  TableOptions,
+  IdType,
+} from 'react-table';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { Icon, useStyles2 } from '@grafana/ui';
+import { isTruthy } from 'app/core/utils/types';
+
+import { EXPANDER_CELL_ID, getColumns } from './utils';
+
+const getStyles = (theme: GrafanaTheme2) => ({
+  table: css`
+    border-radius: ${theme.shape.borderRadius()};
+    border: solid 1px ${theme.colors.border.weak};
+    background-color: ${theme.colors.background.secondary};
+    width: 100%;
+    td,
+    th {
+      padding: ${theme.spacing(1)};
+      min-width: ${theme.spacing(3)};
+    }
+  `,
+  th: css`
+    position: relative;
+  `,
+  sortIcon: css`
+    position: absolute;
+    width: 16px;
+    overflow: hidden;
+  `,
```

Comment:
What's the use case for relative/absolute positioning?

Reply:
probably a leftover, will remove 👍